Makefile

```diff
@@ -131,9 +131,6 @@ run-integ-tests: test-registry gremlin container-health-check-image run-sudo-tes
 run-sudo-tests:
 	sudo -E ${GOTEST} -tags sudo -timeout=10m ./agent/...
 
-run-functional-tests: testnnp test-registry ecr-execution-role-image telemetry-test-image storage-stats-test-image
```

Comment:
a lot of the stuff in the test-registrty is only used for the functional tests -- you can delete a lot of that and stuff in ./misc too

Reply:
I'll create separate PR for cleanup, there's a lot that we don't use like upload-images and replicate-images that need a bit of fine combing through.